clr_debug_daemon.service

```diff
@@ -5,6 +5,8 @@ DefaultDependencies=no
 [Service]
 Type=simple
 ExecStart=/usr/bin/clr_debug_daemon
+# Uncomment to use custom URLs, space separated
```

Comment:
add the environment variable here is not needed, you can add it using a conf file for systemd, probably in /usr/lib/systemd/system/clr_debug_daemon.service.d/env.conf

Reply:
Sure, as long as the functionality is documented somewhere. Since there is no manpage, either the .service or .conf file would be the next best place. In both cases the comments will show up in "systemctl cat".

Reply:
this is documented in https://www.freedesktop.org/software/systemd/man/systemd.unit.html
I think is the best way to provide that environment variable as a vendor, it allows other vendors to change the environment variable without modify the .service file.
other way is use the parameter EnvironmentFile= to create an environment file if it is needed which is documented in
https://www.freedesktop.org/software/systemd/man/systemd.service.html

Reply:
Actually my concern is on the documentation of CLR_DEBUGINFO_URLS itself, if not for us writing it somewhere users would not know that this functionality exists. But it is now so it's all good.